tests/integration/gql/test_responses.py

```diff
@@ -73,6 +74,7 @@ def test_get_gql_response(client, create_experiment, create_ensemble):
     r = client.gql_execute(GET_UNIQUE_RESPONSES, variable_values={"id": ensemble_id})
     for response in r["data"]["ensemble"]["uniqueResponses"]:
         assert response["name"] in RESPONSE_NAMES
+        assert response["recordType"] == "F64_MATRIX"
```

Comment:
Without knowing the details of how `F64_MATRIX` is used in `ert-storage`, I feel like we are leaking implementation details from storage here when we expose the `F64_MATRIX` as part of the API. Especially in combination with https://github.com/equinor/ert/pull/2782, where we use this information without any abstraction in the engine module.

Also why are we only exposing it through the graphql API? :thinking:

Reply:
> Without knowing the details of how `F64_MATRIX` is used in `ert-storage`, I feel like we are leaking implementation details from storage here when we expose the `F64_MATRIX` as part of the API. Especially in combination with [equinor/ert#2782](https://github.com/equinor/ert/pull/2782), where we use this information without any abstraction in the engine module.

I've created an issue that suggest a better naming of record_types #198 
> 
> Also why are we only exposing it through the graphql API? 🤔

I agree, but how such a rest endpoint would look like? Are we to support each record in a similar way to `/userdata/? Graphql provides an easy way to link record_type attribute to each query containing response implicitly... But we can create an issue  on this I think.

Reply:
> Also why are we only exposing it through the graphql API? :thinking:

We've decided that ERT Storage is a GraphQL API with a REST API only for uploading and downloading records. There is difficulty in implementing this at the moment, however, seeing as it requires simultaneous PRs in ERT and ERT Storage. I have made a PR in ERT asking to put an upper version limit so that this may be implemented: https://github.com/equinor/ert/pull/2815